### ISSUE TITLE
Logging when someone expands their claim

### ISSUE THREAD

**Issue by plasticblues** [GriefPrevention/GriefPrevention#84]
I'm suggesting a system implemented to log when a person expands their claim with the following information:

Date boundaries expanded
Where the original claim borders were
Where they are now.

I'd also like to suggest an entry of when the claim was initially created as I believe this currently does not exist.

Thank you so much for looking this over. :)

**Comment by plasticblues** [comment]
Sweet thank you Robo :)

**Comment by bigpresh** [comment]
Yeah, I agree that this would make a lot of sense, so that if someone is complaining that stuff was stolen and "it's always been claimed", a keen admin could use those logs to prove "No, actually, your claim originally didn't cover that area; you later expanded it on $date to cover it".

I wonder if that shouldn't be a plugin that listens for GP hooks, though, so it could be a bit more customisable.

What would be *really* slick would be if someone had the time to write some form of adaptor plugin that would listen for said hooks and shove the data at LogBlock, so LB lookups could cover when a given block was claimed/unclaimed too :)
